### DISCUSSION TITLE
How to build for both mac and windows?

### DISCUSSION THREAD

**Discussion by Nikjee** [quasarframework/quasar#15961] [CLI - Electron mode]
I probably can't understand how extra options in CLI work but i cant figure out how to build for both mac and windows at the same time.

Currently i have to execute commands
quasar build electron --target win --arch x64 --publish always
quasar build electron --target mac --arch arm64 --publish always

I set up in quasar config target and arch for both mac and win but it only build for mac if i specify any other machine like so

quasar build -m electron --target win mac 

Im for sure doing it wrong, can someone help me figure it out please?

quasar.conf.js

` 
electron: {

      inspectPort: 5858,

      bundler: "builder", 

      builder: {
        appId: "...",
        publish: [
          {
            provider: "github",
            repo: "...",
            owner: "...",
            token: ...
            private: true,
          },
        ],
        mac: {
          icon: "src-electron/icons/icon.icns",
          target: [
            {
              target: "default",
              arch: ["universal"],
            },
          ],
        },
        nsis: {
          installerIcon: "src-electron/icons/icon.ico",
          uninstallerIcon: "src-electron/icons/icon.ico",
        },
        win: {
          icon: "src-electron/icons/icon.ico",
          target: [
            {
              target: "nsis",
              arch: ["x64"],
            },
          ],
          publisherName: "reg",
          forceCodeSigning: false,
          verifyUpdateCodeSignature: false,
        },
      },
    },
`

**Comment by suleimanas** [comment]
just an idea, how about using scripts in your `package.json`
```
"scripts": {
    "build:win": "...",
    "build:mac": "...",
    "build": "npm run build:win && npm run build:mac"
},
```

**Reply by Nikjee** [reply]
Yes i am using this because its obvious but im asking if i could do this with one line like just pass multiple args in CLI
